config/detekt/detekt.yml

```diff
@@ -40,6 +40,7 @@ comments:
   UndocumentedPublicClass:
     active: true
     excludes: ['**/test/**', '**/sample/**']
+    ignoreDefaultCompanionObject: true
```

Comment:
I don’t see any value in having documentation on all default (i.e. unnamed) `companion object`s so I’ve disabled this check. Would you agree? @bocops

Reply:
Definitely, agreed.